openquake/hazardlib/tests/source/complex_fault_test.py

```diff
@@ -334,3 +336,59 @@ def test_varying_width(self):
         self.assertEqual(bl, (slice(1, 4), slice(0, 2)))
         self.assertEqual(bm, (slice(1, 4), slice(1, 3)))
         self.assertEqual(br, (slice(1, 4), slice(2, 4)))
+
+
+class ModifyComplexFaultGeometryTestCase(unittest.TestCase):
+    """
+
+    """
+    def setUp(self):
+        top_edge_1 = Line([Point(30.0, 30.0, 1.0), Point(31.0, 30.0, 1.0)])
+        bottom_edge_1 = Line([Point(29.7, 29.9, 30.0),
+                              Point(31.3, 29.9, 32.0)])
+        self.edges = [top_edge_1, bottom_edge_1]
+        self.mfd = EvenlyDiscretizedMFD(7.0, 0.1, [1.0])
+        self.aspect = 1.0
+        self.spacing = 5.0
+        self.rake = 90.
+
+    def _make_source(self, edges):
+        source_id = name = 'test-source'
+        trt = "Subduction Interface"
+        tom = PoissonTOM(50.0)
+        magnitude_scaling_relationship = PeerMSR()
+        cfs = ComplexFaultSource(
+            source_id, name, trt, self.mfd, self.spacing,
+            magnitude_scaling_relationship, self.aspect, tom,
+            edges, self.rake
+        )
+        assert_pickleable(cfs)
```

Comment:
Why the `assert_pickleable`? It should be guaranteed, IIRC there are tests for that in some other part of the code.